lib/logstash/outputs/elasticsearch/http_client.rb

```diff
@@ -135,12 +136,16 @@ def join_bulk_responses(bulk_responses)
     def bulk_send(body_stream)
       params = http_compression ? {:headers => {"Content-Encoding" => "gzip"}} : {}
       # Discard the URL 
+      @metric.increment(:bulk_requests)
+      @metric.increment(:concurrent_bulk_requests)
```

Comment:
If the number of concurrent requests is consistently equal to the number of workers then it can hint that back pressure will happen  in the pipeline.
This number can't be extracted from the number of requests. It may be estimated by taking into account the millis spent doing requests and wallclock time, but seems far fetched.

Also there was a request in 
https://github.com/elastic/logstash/issues/6544 for this too